python/lsst/ip/isr/fringe.py

```diff
@@ -227,11 +310,23 @@ def measureExposure(self, exposure, positions, title="Fringe"):
 
     @timeMethod
     def solve(self, science, fringes):
-        """Solve (with iterative clipping) for the scale factors
-
-        @param science     Array of science exposure fringe amplitudes
-        @param fringes     Array of arrays of fringe frame fringe amplitudes
-        @return Array of scale factors for the fringe frames
+        """Solve (with iterative clipping) for the scale factors.
```

Comment:
Not necessary but "Solve for the scale factors using iterative clipping" would be easier to read.